service-workers/service-worker/no-dynamic-import-in-module.any.js

```diff
@@ -0,0 +1,4 @@
+// META: global=serviceworker-module
+
+import './basic-module-2.js';
```

Comment:
Does this have any purpose? Might be worth adding a comment if it does.

Reply:
It has purpose - I'll add a comment